src/sidebar/components/Excerpt.tsx

```diff
@@ -117,10 +135,9 @@ function Excerpt({
   }, [collapsedHeight, onCollapsibleChanged, overflowThreshold]);
 
   useLayoutEffect(() => {
-    const cleanup = observeElementSize(
-      contentElement.current,
-      updateContentHeight
-    );
+    const cleanup = !contentElement.current
+      ? noop
+      : observeElementSize(contentElement.current, updateContentHeight);
```

Comment:
Again here, I think you can assert the presence of `contentElement.current!`. @robertknight can you sanity-check me?

Reply:
Yes, with a small risk. Currently it is always the case that `contentElement.current` will non-null when this effect is run, since the associated element is unconditionally rendered. The slight risk here is that someone introduces an early-return in the component in some later refactoring, and fails to update `contentElement.current!` assertions. I wonder if any of the linters we use are smart enough to be able to catch that issue.

Reply:
I don't know if there would be a linting/static check capable of detecting that you have used `!` on something that it's actually going to be null at runtime, because precisely the purpose of `!` is silencing those kind of inspections.

I would usually try to do explicit checks as much as possible, even if they look verbose and we "know" that's never going to be really null, because that's the safest approach.

But up to you two. I'm open to use `contentElement.current!` if you think it's the right way to go.

Reply:
I have switched to `contentElement.current!` for now.